Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -769,6 +769,11 @@ theorem lintegral_indicator (f : α → ℝ≥0∞) {s : Set α} (hs : Measurabl
   refine ⟨⟨φ.restrict s, fun x => ?_⟩, le_rfl⟩
   simp [hφ x, hs, indicator_le_indicator]
 
+lemma setLintegral_indicator (f : α → ℝ≥0∞) {s t : Set α} (hs : MeasurableSet s)
+    (ht : MeasurableSet t) : ∫⁻ a in t, s.indicator f a ∂μ = ∫⁻ a in s ∩ t, f a ∂μ := by
```

Comment:
I think that measurability of one of the sets should be enough. E.g., you can `rw [lintegral_indicator, Measure.restrict_restrict]` (or `restrict_restrict'` - not sure). Also, it should be true if the set is only null measurable.

Reply:
Hello Yury,

I just made changes as you suggested. Thanks!